.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           - 2.3
         env:
           -
-            DIFF_LCS_VERSION: "> 1.4.3"
+            BOGUS: ""
```

Comment:
Tried hard to remove the whole this `env` block to no avail.

Reply:
Its the `matrix.env` call, I don't think theres an `if` in the syntax :joy: I'd just set it to ruby head false or something

Reply:
Right! I've tried:
 - removing it
 - setting it to an empty array `[]`
 - setting it to an empty hash `{}`
 - leaving a `-`

I give up 😆
ruby head false would create a wrong impression that it's there for a good reason, while this non-ambiguously says "some cruft" 😄

Reply:
It needs to be either something real, or something explicit, however this also needs back porting to rspec-dev, so it can be addressed later.